### ISSUE TITLE
Main package not found

### ISSUE THREAD

**Issue by b1zzu** [unbroken-dome/gradle-testsets-plugin#60]
Hi,
I have migrate my `build.gradle` to Kotlin DSL in this way:

```
plugins {
    java
    idea

   id("org.unbroken-dome.test-sets") version "2.0.2"
}

testSets {
    create("integrationTest")
}

tasks.test {
    useJUnitPlatform()
}

tasks.named<Test>("integrationTest") {
    useJUnitPlatform()

}
```
under `src/main/java` i have my source code, under `src/test/java` my unit tests and under `src/integrationTest/java` my integration tests.

I execute my unit tests `./gradlew  test` and they work as expected but if I execute the integration tests `./gradlew integrationTest` the build fail during the `compileIntegrationTestJava` task with this error: 

```

[...]/src/integrationTest/java/com/example/server/repositories/QueueManagerRepositoryIntegrationTest.java:3: error: package com.example.server.entities does not exist
import com.example.server.entities.QueueManager;
                                     ^
```
I'm using gradle 5.0 and in groovy it work perfectly.

Am I doing something wrong?

**Comment by tkrullmann** [comment]
This should be fixed with 2.0.3. Can you please try?

**Comment by tolleiv** [comment]
Hi - I had a similar issue with version 2.0.2 and the upgrade to 2.0.3 resolved it for me. Thx for fixing it :)

**Comment by b1zzu** [comment]
@tkrullmann yes this issue is solved with 2.0.3
